tests/system/Jenkinsfile

```diff
@@ -9,15 +9,16 @@ def config = jobConfig {
 
 def job = {
     configureGitSSH("github/confluent_jenkins", "private_key")
+    def mavenSettingsFile = "/home/jenkins/.m2/settings.xml"
```

Comment:
Is this always going to be the case? Would there be a better location in this file for this constant?

Reply:
Honestly I don't feel super good about it being in a non-temp location so I might want to change it to WORKSPACE_TMP regardless of what it was before..
As for location in this file, it'll have to stay within the `job {}` closure if we want to use/anticipate there being any Jenkins env variables